src/components/mixins/SelectAssetMixin.ts

```diff
@@ -45,26 +48,16 @@ export default class SelectAsset extends Mixins(mixins.DialogMixin, AssetsSearch
     };
   }
 
-  public getAssetsWithBalances({
-    assets,
-    accountAssetsAddressTable,
-    excludeAsset,
-  }: {
-    assets: Array<Asset | RegisteredAccountAsset>;
-    accountAssetsAddressTable: WALLET_TYPES.AccountAssetsTable;
-    excludeAsset?: Asset | AccountAsset;
-  }): Array<AccountAsset | RegisteredAccountAsset> {
-    return assets.reduce((result: Array<AccountAsset>, item) => {
-      if (!item || (excludeAsset && item.address === excludeAsset.address)) return result;
-
-      const accountAsset = accountAssetsAddressTable[item.address];
-      const balance = accountAsset?.balance;
-      const prepared = {
-        ...item,
-        balance,
-      };
-
-      return [...result, prepared];
+  public getAssetsWithBalances(addresses: string[], excludeAddress = ''): Array<RegisteredAccountAsset> {
```

Comment:
Do we really need it? I guess we'll have these mappings right on the component during the network selection

Reply:
got it